CondCore/Utilities/python/tier0.py

```diff
@@ -123,7 +133,8 @@ def _queryTier0DataSvc( self, url ):
            logging.error(msg)
 
            time.sleep(10)
-           cmd = '/usr/bin/curl -k -L --user-agent "%s" %s --connect-timeout %i --retry %i %s %s ' % (userAgent, proxy, self._timeOut, self._retries, "-v", url)
+           cmd = '/usr/bin/curl -k -L --user-agent "%s" %s --connect-timeout %i --retry %i %s %s %s ' % \
+              (userAgent, proxy, self._timeOut, self._retries, "-v", url, certs)
```

Comment:
I would split the lines in the same way as at [L118-119](https://github.com/cms-sw/cmssw/pull/45779/files#diff-5690ef6a21b100d1e130e497d0579f614f574426ed3c9442b8c6cf7cbcafad77R118-R119) here above, so that it is evident that you are repeating the very same command
```suggestion
        cmd = '/usr/bin/curl -k -L --user-agent "%s" %s --connect-timeout %i --retry %i %s %s %s' \
            % (userAgent, proxy, self._timeOut, self._retries, debug, url, certs)
```